src/components/menu.component.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+import Icon from 'react-native-vector-icons/Ionicons';
+
+import React, {Component, NavigatorIOS, TabBarIOS, View, AlertIOS,} from 'react-native';
```

Comment:
That's smart since I accidentally pushed to develop yesterday and it saved my ass.